syntax = "proto3";

message LocationMessage {
  int64 id = 1;
  int64 person_id = 2;
  string longitude = 3;
  string latitude = 4;
  string creation_time = 5;
}

message LocationId {
  int64 id = 1;
}

message PersonId {
  int64 id = 1;
}

message LocationMessageList {
  repeated LocationMessage locations = 1;
}


service LocationService {
    rpc GetById(LocationId) returns (LocationMessage);
    rpc GetByPersonId(PersonId) returns (LocationMessageList);
}
